reana_client/cli/outputs.py

```diff
@@ -162,10 +162,18 @@ def outputs_download(ctx, user, organization, workflow, file_, output_folder):
                                                              file_name)
             logging.info('{0} binary file downloaded ... writing to {1}'.
                          format(file_name, output_folder))
-            with open(os.path.join(output_folder, file_name), 'wb') as f:
+
+            outputs_file_path = os.path.join(output_folder, file_name)
+            if not os.path.exists(os.path.dirname(outputs_file_path)):
+                    os.makedirs(os.path.dirname(outputs_file_path))
+
+            with open(outputs_file_path, 'wb') as f:
                 f.write(binary_file)
             click.echo('File {0} downloaded to {1}'.format(file_name,
                                                            output_folder))
+        except OSError as e:
```

Comment:
This `exists_ok` is python 3 (see [here](https://docs.python.org/3/library/os.html#os.makedirs)) but makedirs in python 2 would also through an `OSError` (see [here](https://docs.python.org/2/library/os.html#os.makedirs)). I have tried to `reana-client outputs download fitdata/plot.png` in two cases: existing `outputs/fitdata/plot.png` and removing the file but keeping the directory so `outputs/fitdata` exists. In both cases it works and overwrites. Just in case we add an OSError except block for writing errors.